apps/framr-web/lib/services/core/modules/tools/ToolsService.ts

```diff
@@ -0,0 +1,128 @@
+import { CreateTool, MWDTool } from '../../../../types';
+import { ToolEnum } from '../../../../types/enums';
+import { FramrServiceError } from '../../../libs/errors';
+import { EventBus, EventBusChannelStatus } from '../../../libs/event-bus';
+import { IDBFactory, StoreRecord } from '../../../libs/idb';
+import { IDBConnection } from '../../db/IDBConnection';
+import { FramrDBSchema } from '../../db/schema';
+
+import { ToolInterface, ToolsEventChannel } from './ToolInterface';
+
+export class ToolsService implements ToolInterface {
+  private readonly eventBus: EventBus;
+  private readonly database: IDBFactory<FramrDBSchema>;
+
```

Comment:
Add  constant attribute that reflects the name of the store object that is been manipulate and replace all matching occurrences 
```suggestion
   private readonly STORE_NAME = "tools"
```